src/WindowsTracing/MockTracerListener.h

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef WINDOWS_TRACING_MOCK_TRACER_LISTENER_H_
+#define WINDOWS_TRACING_MOCK_TRACER_LISTENER_H_
+
+#include <vector>
+
+#include "WindowsTracing/TracerListener.h"
+#include "capture.pb.h"
+namespace orbit_windows_tracing {
+class MockTracerListener : public orbit_windows_tracing::TracerListener {
```

Comment:
Personal preference: You can move that mock class into the test directly.

Reply:
Yep, thanks, done.